tests/test_signed.rs

```diff
@@ -0,0 +1,265 @@
+#![feature(plugin, custom_attribute)]
+#![plugin(rustproof)]
+#![allow(dead_code)]
+#![allow(unused_attributes)]
+fn main() { }
+
+// FIXME: some preconditions are unnecisarilly restricitve ie add_five_i32_invalid does not need x: i32 >= i32::MIN + 5i32
+
+// * * *
+// Integer Add Tests
+// * * *
+
+// Tests signed integer overflow for 32 bit integers
+// Should be invalid
+#[condition(pre="(x: i32 > 0) && (x: i32 < 10)", post="return: (x: i32 > 30)")]
```

Comment:
@salterm disreguard. This is caught by the parser, but tests pass regardless. I will fix up the tests.